### ISSUE TITLE
Ability to specify notification color in slack

### ISSUE THREAD

**Issue by M0nsieurChat** [cloudfoundry-community/slack-notification-resource#63]
Hello !

First of all, thanks for your amazing work !

Would it be possible to introduce a new attribute `color:` so that we could specify which color would be applied to the text notification inside Slack ?

I think this is possible because our other slack integrations will apply different colors for different notifications statuses

![screenshot from 2019-01-21 12-35-43](https://user-images.githubusercontent.com/4733617/51472275-4fe27400-1d79-11e9-8c34-0514813beab8.png)
![screenshot from 2019-01-21 12-35-28](https://user-images.githubusercontent.com/4733617/51472276-4fe27400-1d79-11e9-93cd-195afa005cbe.png)

That was we could differentiate on_success and on_failure notifications and make more clear which builds failed and which builds succeed

Thanks,

**Comment by jhunt** [comment]
Definitely!  Is that something you would be interested in working on / submitting a PR for?

**Comment by M0nsieurChat** [comment]
Looking at the slack documentation, it seems this "color" is only achievable through attachments : https://api.slack.com/docs/message-attachments

Fortunately, there is an `attachments:` parameter so I might be able to reproduce this without altering the container resource.
